release-engineering/platforms.md

```diff
@@ -43,12 +43,14 @@ platform.
 
 ### Tier 3
 
-Tier 3 platforms are those which the Kubernetes codebase has functional support
-for, but which are not built or tested automatically. This means the binary
-artifacts may not work as intended.
+Tier 3 platforms are those that have been demonstrated to work and have a
+documented build process that is available to anyone in the community. There is
+no guarantee that builds will continue to work, and the platform may be dropped
+if documentation is not maintained.
 
 - Official builds are not available.
 - Automated testing is not setup.
+- Documentation on how to build for the platform is available.
```

Comment:
I am trying to decide if this documentation belongs in sig-release/release-engineering or in https://github.com/kubernetes/community/tree/master/contributors/devel/sig-architecture. I would like to keep as much of this information located together as possible, but we already have the beginning of guide in k/community. Ultimately, it seems like this primarily falls under release-engineering in terms of shepherding people through the process, and we also need to make sure release-engineering has knowledge of how to build all platforms, so I am somewhat inclined to move the guide here, point folks to it, and have Tier 3 build docs also live here.

Thoughts? @dims @saschagrunert

Reply:
release-engineering sounds good to me!

Reply:
I agree this should live here.